CODEOWNERS

```diff
@@ -1 +1,5 @@
 * @dcos/cluster-ops-fnd
```

Comment:
can this not be removed?

Reply:
We have a check that require person from team to approve